apps/src/applab/applab.js

```diff
@@ -1139,7 +1139,10 @@ Applab.execute = function() {
     makerToolkit
       .connect({
         interpreter: Applab.JSInterpreter,
-        onDisconnect: () => studioApp().resetButtonClick()
+        onDisconnect: () => {
+          studioApp().resetButtonClick();
+          makerToolkit.disconnectBoard();
```

Comment:
This reference back to `makerToolkit.disconnectBoard()` probably shouldn't live in applab.js - it's logic internal to Maker Toolkit that we wouldn't want to duplicate if we ever add maker to gamelab.

Instead, I'd wrap the 'client' `onDisconnect` where we attach it to the board controller with a callback that includes maker's own internal cleanup logic:

https://github.com/code-dot-org/code-dot-org/blob/f90bd8360531aec6137e4655d10040cf0f862694/apps/src/lib/kits/maker/toolkit.js#L92-L94